book/models.py

```diff
@@ -17,7 +17,8 @@ class Book(models.Model):
     inventory = models.PositiveIntegerField()
     daily_fee = models.DecimalField(
         max_digits=5,
-        decimal_places=2
+        decimal_places=2,
+        null=True
```

Comment:
I also can't really see what this `null=True` is for. Can you clarify this?

Reply:
hmmm you right 👍

Reply:
Should this line be deleted then?

Reply:
yes

Reply:
i'm trying 😊